addons/overpressure/functions/fnc_fireOverpressureZone.sqf

```diff
@@ -28,18 +27,23 @@ private ["_position", "_direction"];
 _position = getPosASL _projectile;
 _direction = vectorDir _projectile;
 
-private ["_dangerZoneAngle", "_dangerZoneRange", "_dangerZoneDamage"];
+private ["_var", "_varName", "_dangerZoneAngle", "_dangerZoneRange", "_dangerZoneDamage"];
 
-_dangerZoneAngle = getNumber (configFile >> "CfgWeapons" >> _weapon >> QGVAR(angle)) / 2;
-_dangerZoneRange = getNumber (configFile >> "CfgWeapons" >> _weapon >> QGVAR(range));
-_dangerZoneDamage = getNumber (configFile >> "CfgWeapons" >> _weapon >> QGVAR(damage));
+// Bake Variablen Name and Check if the Variable Exist else call the Cache Function
```

Comment:
`// Bake variable name and check if the variable exists, call the caching function otherwise`